release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -444,6 +444,39 @@ For more information, see the following sections:
 [id="ocp-4-15-storage"]
 === Storage
 
+[id="ocp-4.15-storage-recovering-vgs-from-prev-installation"]
+==== Recovering volume groups from the previous {lvms} installation
+With this release, `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates that volume group in the current {lvms} installation. This simplifies the process of data recovery through the re-installation of {lvms}.
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-volume-cr_persistent-storage-local[Provisioning local volumes by using the Local Storage Operator].
+
+[id="ocp-4.15-storage-support-for-wiping-the-devices"]
+==== Support for wiping the devices
```

Comment:
I see that LSO and LVMS release notes are bundled inside Storage section. Would it make sense to separate them into their titles under Storage? If not, we should use the LVMS product name in every feature that is related to LVMS like this one.

Reply:
Hi @suleymanakbas91 
I agree with you. I have included "LVM Storage" in all the section headings. 
The other sections in the release note do not have sections based on Operators. Hence, including the Operator name in the section heading would be best to maintain consistency.

Reply:
QQ: should we for an optional operator even add this to the central release notes? Probably not right? Because we have separate release cadence.

Reply:
We should have it in the central release notes as LVMS is aligned with OCP release.